.github/workflows/matrix-update.yml

```
@@ -0,0 +1,68 @@
+name: Compatibility matrix update
+on:
+  workflow_call: {}
+
+env:
+  NODE_VERSION: "20.11.0"
+  PNPM_VERSION: "8.15.1"
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2.2.4
+        with:
+          version: ${{ env.PNPM_VERSION }}
+
+      - name: Setup Node.js v18
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Install pnpm Dependencies
+        run: pnpm install --frozen-lockfile --filter ./tools/compatibility-matrix-automation
+
+      - name: Download Output Files
+        uses: actions/download-artifact@v3
+        with:
+          name: out
+          path: out
+
+      - name: Run Update Script
+        run: |
+          cd tools/compatibility-matrix-automation
+          TEST_OUTPUT_FOLDER=../../out npx update-matrix
+
+      - name: Create a PR
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: ${{ secrets.PROJEN_GITHUB_TOKEN }}
+          title: "chore(docs): compatibility matrix update"
+          branch: "github-actions/updating-matrix"
+          base: main
+          add-paths: docs/**
+          commit-message: |-
+            chore(docs): update compatibility matrix
+
+            [Workflow Run]: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
```

Is the workflow run meant to be a link? Also I think the first line can be removed, maybe replaced with a link to the docsite

```suggestion
            Updates the [Compatibility Matrix](https://www.winglang.io/docs/standard-library/compatibility-matrix)

            [Workflow Run](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})
```